### PR TITLE
Rename StageExtras to LessonExtras

### PR DESCRIPTION
**Pull request by dmcavoy**
Rename StageExtras to LessonExtras in react code.

### CODE REVIEW

File: apps/src/code-studio/components/lessonExtras/LessonExtras.jsx [hunk 1]
```diff
@@ -46,7 +46,7 @@ export default class StageExtras extends React.Component {
 
     return (
       <div>
-        {showStageExtrasWarning && sectionId && <StageExtrasNotification />}
+        {showStageExtrasWarning && sectionId && <LessonExtrasNotification />}
```
Comment (Erin007) [inline apps/src/code-studio/components/lessonExtras/LessonExtras.jsx]: change the `showStageExtrasWarning` variable too?

File: apps/src/code-studio/components/lessonExtras/LessonExtras.story.jsx [hunk 2]
```diff
@@ -23,11 +23,11 @@ function configureStore() {
...
   const store = configureStore();
-  storybook.storiesOf('StageExtras', module).add(
+  storybook.storiesOf('LessonExtras', module).add(
     'default',
-    withInfo('This is the StageExtras component.')(() => (
+    withInfo('This is the LessonExtras component.')(() => (
       <Provider store={store}>
-        <StageExtras
+        <LessonExtras
           stageNumber={1}
```
Comment (Erin007) [inline apps/src/code-studio/components/lessonExtras/LessonExtras.story.jsx]: lessonNumber? Or is the idea just to do component/file names now?

**Review by Erin007 — COMMENTED** [review]
Is this just about renaming files and components now?  If so, looks good; if we've time to update prop and variable names within these components - even better!

**Comment by dmcavoy** [comment]
> Is this just about renaming files and components now? If so, looks good; if we've time to update prop and variable names within these components - even better!

@Erin007 Renamed the props for LessonExtras and LessonExtrasProgressBubble

File: apps/test/unit/code-studio/components/progress/LessonProgressTest.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {assert} from 'chai';
+import {UnconnectedLessonProgress as LessonProgress} from '@cdo/apps/code-studio/components/progress/LessonProgress';
+import {LevelStatus} from '@cdo/apps/util/sharedConstants';
+import {shallow} from 'enzyme';
+
+describe('LessonProgress', () => {
```
Comment (davidsbailey) [inline apps/test/unit/code-studio/components/progress/LessonProgressTest.js:7]: Thank you for breaking these renames into individual commits! It's odd that github doesn't recognize this file as having been renamed.

**Review by davidsbailey — APPROVED** [review]
Looks good, @dmcavoy ! Thanks for doing this!

**Pull request merged**